src/Hospital/Schedule/Repository/AppointmentFileRepository.cs

```diff
@@ -34,11 +34,11 @@ public List<Appointment> GetAll()
 
         public Boolean Save(Appointment newAppointment)
         {
-            newAppointment.AppointentId = GenerateNextId();
+            newAppointment.Id = GenerateNextId();
```

Comment:
ne koristiti rucno generisanje id-ja. baza radi to

Reply:
To je ostalo iz json fajlova, ne generisem rucno za bazu